conf/defaults.ini

```diff
@@ -397,6 +397,7 @@ global_session = -1
 [alerting]
 # Makes it possible to turn off alert rule execution.
 execute_alerts = true
+show_alerting_ui = true
```

Comment:
if the option hides ui making it accessible, maybe just name it "enabled"   (And AlertingEnabled in code)